classes/person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Getters and Setters for `@rentals` should be present, rentals is an `attr_accessor`, please kindly change it 🙃 🙃